packages/react/src/categories/hooks/__tests__/useCategories.test.tsx

```diff
@@ -44,126 +38,73 @@ describe('useCategories', () => {
   afterEach(cleanup);
 
   it('should return values correctly with initial state', () => {
-    const current = getRenderedHook();
+    const current = getRenderedHook(mockCategoriesState);
 
     expect(current).toStrictEqual({
-      areCategoriesFetched: expect.any(Boolean),
-      areCategoriesLoading: expect.any(Boolean),
-      areTopCategoriesFetched: expect.any(Boolean),
-      areTopCategoriesLoading: expect.any(Boolean),
-      categories: expect.any(Array),
-      categoriesError: expect.any(Object),
-      fetchCategories: expect.any(Function),
-      fetchTopCategories: expect.any(Function),
-      getCategory: expect.any(Function),
-      getRootCategory: expect.any(Function),
-      resetCategoriesState: expect.any(Function),
-      topCategories: expect.any(Array),
-      topCategoriesError: expect.any(Object),
+      isFetched: true,
+      isLoading: false,
+      data: Object.values(mockCategoriesState.entities.categories),
+      error: null,
+      actions: {
+        fetch: expect.any(Function),
+        reset: expect.any(Function),
+      },
     });
   });
 
-  it('should render in categories loading state', () => {
-    const { areCategoriesLoading } = getRenderedHook(
-      mockCategoriesLoadingState,
-    );
+  it('should render loading state', () => {
```

Comment:
```suggestion
  it('should return the loading state correctly', () => {
```

Please, check the other occurrences as well.